plugins/lookup/passwordstore.py

```diff
@@ -60,6 +60,10 @@
         description: use alphanumeric characters.
         type: bool
         default: 'no'
+      ignore_missing:
+        description: Do not report error upon non-existent password file
+        type: bool
+        default: 'no'
```

Comment:
```suggestion
        default: false
```
(The others should be changed as well, but that probably should be done in another PR.)

Reply:
Ok, for the sake of consistency, I propose to leave that on 'no' and change all values in a subsequent PR (I will gladly do that PR as well).

Reply:
We really shouldn't add more bad data. That's a lot more important IMO than consistency with existing bad data. Also not fixing it here makes it impossible to auto-backport the follow-up PR to older branches, since there will be conflicts because of this new option.